code/modules/tgui/states/default.dm

```diff
@@ -16,6 +16,9 @@ GLOBAL_DATUM_INIT(default_state, /datum/ui_state/default, new)
 /mob/proc/default_can_use_topic(src_object)
 	return UI_CLOSE // Don't allow interaction by default.
 
+/mob/dead/observer/default_can_use_topic(src_object)
+	return UI_UPDATE
```

Comment:
won't this allow observers to do actions on any UI?

Reply:
https://github.com/BeeStation/BeeStation-Hornet/blob/master/code/__DEFINES/tgui.dm#L3-L4
```
/// Orange eye; updates but is not interactive
#define UI_UPDATE 1
```

Reply:
I am confused why this is added then, what does it do that wasn't already possible

Reply:
Honestly tgui code confuses me, this is just redundancy so ghosts can read UIs by default anyways in most situations?

Reply:
This shouldnt be needed due to code in ui_status which already adds this if the client is within range of the object. Note that if the object is a datum this code won't work though.
If you want to add support to ghosts viewing datum UIs, update ui_status instead of adding this.

This adds the unintended behaviour of ghosts being able to view UIs that aren't on the screen.